tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended", "tslint-config-prettier"],
+  "jsRules": {},
+  "rulesDirectory": [],
+  "rules": {
+    "only-arrow-functions": false,
+    "no-bitwise": false,
+    "class-name": false,
+    "interface-name": false,
+    "member-access": false,
+    "object-literal-sort-keys": false
```

Comment:
These rules were added so that I didn't have to refactor code unnecessarily. I would recommend at the minimum turning on `only-arrow-functions` -- `function` brings with it `this` and all of its baggage, and so arrow functions tend to simplify and make code more functional. The other values should also be turned on, but will require light code modifications.